app/apps/routes.js

```diff
@@ -2,10 +2,11 @@ const handlers = require('./handlers');
 
 
 module.exports = [
-  { name: 'new', pattern: '/apps/new', handler: handlers.new_app },
-  { name: 'list', pattern: '/apps', handler: handlers.list_apps },
-  { name: 'create', method: 'POST', pattern: '/apps/create', handler: handlers.create_app },
-  { name: 'list_user_apps', pattern: '/user/:id/apps', handler: handlers.list_user_apps },
-  { name: 'details', pattern: '/apps/:id', handler: handlers.app_details },
-  { name: 'delete', method: 'POST', pattern: '/apps/:id/delete', handler: handlers.app_delete },
+  { name: 'new', pattern: '/apps/new', handler: handlers.new },
+  { name: 'list', pattern: '/apps', handler: handlers.list },
+  { name: 'create', method: 'POST', pattern: '/apps/create', handler: handlers.create },
+  { name: 'details', pattern: '/apps/:id', handler: handlers.details },
+  { name: 'delete', method: 'POST', pattern: '/apps/:id/delete', handler: handlers.delete },
+  //{ name: 'connect_bucket', method: 'POST', pattern: '/apps/:app_id/buckets', handler: handlers.connect_bucket },
```

Comment:
Don't think we have this endpoint anymore, and also not sure what the `update_bucket` (commented) endpoint would do.

Reply:
They are not supposed to be there, I will remove them